tests/filecheck/dialects/riscv/riscv_assembly_emission.mlir

```diff
@@ -153,4 +153,11 @@
   // CHECK-NEXT: ebreak
   "riscv.ret"() : () -> ()
   // CHECK-NEXT: ret
+  "riscv.directive"() {"directive" = ".space", "value" = "1024"} : () -> ()
+  // CHECK-NEXT: .space 1024
+  "riscv.directive"() ({
+    "riscv.directive"() {"directive" = ".space", "value" = "1024"} : () -> ()
+  }) {"directive" = ".bss"} : () -> ()
+  // CHECK-NEXT: .bss
+  // CHECK-NEXT: .space 1024
```

Comment:
What does the `.space` directive do? Is it possible that this is an ARM directive?